lib/glueby/contract/timestamp.rb

```diff
@@ -55,12 +55,14 @@ def initialize(
         wallet:,
         content:,
         prefix: '',
-        fee_provider: Glueby::Contract::FixedFeeProvider.new
+        fee_provider: Glueby::Contract::FixedFeeProvider.new,
+        digest: 'sha256'
```

Comment:
The digest value is better for symbols than strings like (:sha256, :double_sha256 or :none)

Reply:
OK